### DISCUSSION TITLE
How to delete all data in one go? this Duplicate bug is crazy

### DISCUSSION THREAD

**Discussion by ui404** [ente-io/ente#3671] [Q&A]
Hi, I use Ente Auth for both my phone and desktop applications. However, I have one issue: duplicates. 
I can delete them individually, but it's exhausting because I have many 2FA accounts, and deleting them one by one is time-consuming since you need to long-press each account, click the trash icon, and wait for it to disappear. 

The duplication seems very random; sometimes, it copies the same account in the desktop app. There is no option to delete all at once, as I am trying to do an import again to tidy things up.

**Comment by eric-gitta-moore** [comment]
If you don't have to, consider bitwarden?

**Reply by ui404** [reply]
bitwarden is password manager, you need paid plan to use intergrated authenticator
I rather go with microsoft authenticator than bitwarden but that not the case

I much prefer open source like ente auth that support many platform 
2Fas and Aegis also pretty good but no desktop apps

**Reply by eric-gitta-moore** [reply]
> [okezumi ](/okezumi) [ Oct 12, 2024 ](#discussioncomment-10921500)

I learned that bitwarden has an open source third party version called vaultwarden

https://github.com/dani-garcia/vaultwarden

Yes, there doesn't seem to be a desktop program. Probably for personal reasons, I don't like software with desktop programs, I prefer plugins or web pages in the browser hahaha

**Reply by ui404** [reply]
> Yes, there doesn't seem to be a desktop program. Probably for personal reasons, I don't like software with desktop programs, I prefer plugins or web pages in the browser hahaha

Since I am using a lot of 2FA, not all require a code from the browser; copy and pasting in native desktop apps is much easier task without opening a browser, especially for crypto transactions or withdrawals. gaming apps, require 2FA from within the apps themselves.
